### ISSUE TITLE
Undefined ArrayKey systemRootPath

### ISSUE THREAD

**Issue by ally9696** [WWBN/AVideo#8094]
Warning: Undefined array key "systemRootPath" in C:\laragon\www\AVideo\locale\function.php on line 175

Warning: Undefined array key "systemRootPath" in C:\laragon\www\AVideo\locale\function.php on line 184

Warning: Undefined array key "systemRootPath" in C:\laragon\www\AVideo\locale\function.php on line 192

POPUP Unknown error during installation.

Those error of global variable was not there when installing it first time
When i had installed it first time it successfully installed and it was green and to delete the install directory.
Xampp 2014 PHP7.4

now after i wanted to reinstall it wont work and it gives error on older and new version2021 of xampp and laragon latest. How do i fix it..

i also tried to remove db and avideo files and again follow same process but it still won't work.

**Comment by DanielnetoDotCom** [comment]
are you running the last git pull?

your errors does not match with the latest code

**Comment by ally9696** [comment]
> are you running the last git pull?
> 
> your errors does not match with the latest code

Yes Latest but its working on my 2019 Xampp server but if In use another server "Laragon" then it gives that error, I don't know why.

**Comment by DanielnetoDotCom** [comment]
sorry this is NOT the latest
